File: java/Ejercicio36.java
import java.util.Scanner;

public class Ejercicio36 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
  

        System.out.print("Ingrese el radio (r): ");
        double radio = sc.nextDouble();

        System.out.print("Ingrese el ángulo theta en grados (desde el eje Z): ");
        double thetaGrados = sc.nextDouble();

        System.out.print("Ingrese el ángulo phi en grados (en el plano XY): ");
        double phiGrados = sc.nextDouble();

        double theta = Math.toRadians(thetaGrados);
        double phi = Math.toRadians(phiGrados);

        double x = radio * Math.sin(theta) * Math.cos(phi);
        double y = radio * Math.sin(theta) * Math.sin(phi);
        double z = radio * Math.cos(theta);

        System.out.println("Las coordenadas cartesianas son: (" + x + ", " + y + ", " + z + ")");

        sc.close();
    }
}

File: python/Ejercicio36.py
import math

radio = float(input("Ingrese el radio (r): "))
theta_grados = float(input("Ingrese el ángulo theta en grados (desde el eje Z): "))
phi_grados = float(input("Ingrese el ángulo phi en grados (en el plano XY): "))

theta = math.radians(theta_grados)
phi = math.radians(phi_grados)

x = radio * math.sin(theta) * math.cos(phi)
y = radio * math.sin(theta) * math.sin(phi)
z = radio * math.cos(theta)

print("Las coordenadas cartesianas son: (", x, ",", y, ",", z, ")")